.github/workflows/auto-pr.yml

```
@@ -0,0 +1,64 @@
+  create_pr_to_main:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'uat'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Check for Existing PR to main
+        id: check_main_pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.SECRET_GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:46: Double quote to prevent globbing and word splitting

(shellcheck)